src/pycram/cache_manager.py

```diff
@@ -50,7 +52,8 @@ def delete_cache_dir(self):
 
     def update_cache_dir_with_object(self, path: str, ignore_cached_files: bool,
                                      object_description: 'ObjectDescription', object_name: str,
-                                     scale_mesh: Optional[float] = None) -> str:
+                                     scale_mesh: Optional[float] = None,
+                                     mesh_transform: Optional['Transform'] = None) -> str:
```

Comment:
When you import "from __future__ import annotations" you can use Transform directly without the quotation marks